omod/src/main/webapp/resources/htmlforms/003-HMIS_HIVCare_ARTCard-ClinicalAssessmentPage.xml

```diff
@@ -0,0 +1,1531 @@
+                    }
+
+                    /* Third Line*/
+
+                    if (onThirdLine !== "true") {
+                        stableArray.push(true);
+                    } else {
+                        stableArray.push(false);
+                        instabilityCause += "Current regimen is a third line regimen \n "
+                    }
+
+                    /* Adherence Check*/
+
+                    if (adherence.length &lt;= 0 || adherence.trim().includes("POOR")) {
+                        stableArray.push(false);
+                        instabilityCause += "Adherence is not Good \n "
+                    } else {
+                        stableArray.push(true);
+                    }
+
+                    /* Clinic staging */
+                    if (clinicStaging === "90033" || clinicStaging === "90034" || clinicStaging === "90293" || clinicStaging === "90294") {
+                        stableArray.push(true);
+                    } else {
+                        stableArray.push(false);
+                        instabilityCause += "Clinic stage is neither 1 or 2 \n";
+                    }
+                    var pageCurrentRegimen = jq("#90315").find("select").find(":selected").val();
+                    var pageEncounterDate = jq("#encounterDate").find("input[type=hidden]").val();
+                    var dsdmRegimenCoceptsIds="164976,164977,164978,164979";
+
+                    if (currentRegimen !== "" &amp;&amp; (currentRegimen === pageCurrentRegimen || dsdmRegimenCoceptsIds.trim().includes(pageCurrentRegimen)) &amp;&amp; diff_months(new Date(pageEncounterDate), new Date(regimenStartedDate)) &gt; 12) {
+
+                        stableArray.push(true);
+                    } else {
+                        stableArray.push(false);
+                        instabilityCause += "Patient has not spent more than 12 moths on current Regiment \n "
+                    }
+
+                    if (sputumValue !== "" &amp;&amp; sputumValue !== "664" &amp;&amp; diff_months(new Date(jq("#encounterDate").find("input[type=hidden]").val()), new Date(teatmentStartDate)) &lt; 2) {
+                        stableArray.push(false);
+                        instabilityCause += "Patient has not been on treatment for TB for at least 2 months"
+                    } else {
+                        stableArray.push(true);
+                    }
+                    instabilityCause.trim().replace()
+
+                    if (stableArray.indexOf(false) != -1) {
+                        jq("#patient-categorization select").val("90004");
+                    } else {
+                        jq("#patient-categorization select").val("90003");
+                    }
+
+                }
+                <!-- Enabling and Disabling Prescription of multiple drugs -->
+                //addRemovePrescription();
+
+                $allRows = jq(this).find('.multi-drug');
```

Comment:
@semujju what is the class multi-drug for? Its not clear from reading it

Reply:
This class (multi-drug) has been renamed to drugs-prescribed, Its is attached to a div where more than a single prescription will be represented